File: src/paco/lugares/comer/opendata/chascarentenerife/models/Comentario.java
package paco.lugares.comer.opendata.chascarentenerife.models;

import com.roscopeco.ormdroid.Column;
import com.roscopeco.ormdroid.Entity;
import static com.roscopeco.ormdroid.Query.eql;

public class Comentario extends Entity{

	  @Column(primaryKey=true)
	  public int id;
	  public String usuario, fecha, valoracion, precio, recomendacion, opinion;
	  
	  public Comentario(){
	     this(null, null, null, null, null, null);
      }
	  
	  public Comentario(String usuario, String fecha, String valoracion, String precio, String recomendacion, String opinion){
		  this.usuario = usuario;
		  this.fecha = fecha;
		  this.valoracion = valoracion;
		  this.precio = precio;
		  this.recomendacion = recomendacion;
		  this.opinion = opinion;
	  }


}

File: servidor/models/comentario.js
// User Class
var mongoose = require('mongoose'),  
    Schema = mongoose.Schema;  
  
var comentarioSchema = new Schema({  
    usuario: String,
    fecha: Date, 
    valoracion: { type: Number, default: 0 }, 
    precio: { type: Number, default: 0 },
    recomendacion: String, 
    opinion: String,
    idestablecimiento: String
});  
  
//Export the schema  
module.exports = mongoose.model('Comentario', comentarioSchema); 